tests/e2e/containers/containers.go

```diff
@@ -115,6 +117,16 @@ func (m *Manager) ExecTxCmdWithSuccessString(t *testing.T, chainId string, conta
 	if addGasFlags {
 		allTxArgs = append(allTxArgs, txDefaultGasArgs...)
 	}
+	// Add memo field to every tx
+	// This is done because in E2E, we remove the sequence number ante handler.
+	// This allows for quick throughput of txs, but if two txs are the same (i.e. two CL claims, two bank sends with the same amount),
+	// the sdk treats them as the same tx and errors with code 19 (tx already in mempool). By changing the memo field on every tx,
+	// we ensure that every tx is unique and can be submitted.
+	memo := fmt.Sprintf("--note=%d", memoCounter)
+	allTxArgs = append(allTxArgs, memo)
+	// Increment the counter for the next tx
+	memoCounter++
```

Comment:
I thought we can run txs in parallel? Do we not need to keep `memoCounter` behind mutex?

Reply:
So in order for this to be an issue, the tx happening in parallel would need to be the exact same tx, which is super unlikely (i.e. we would need to have have two separate tests call the same message with the same account at the same time). You are totally right though that it's possible and should probably be mutex'd, im just explaining the probable reason of why I haven't seen any failures yet.

Reply:
Done here https://github.com/osmosis-labs/osmosis/pull/7048/commits/614e81c6651cef5bb264f4b0c9de4491633d4847